internal/docker/builder.go

```diff
@@ -539,6 +539,7 @@ func printPortBindingsOfAllComplementContainers(docker *client.Client, contextSt
 	log.Printf("=============== %s : END ALL COMPLEMENT DOCKER PORT BINDINGS ===============\n\n\n", contextStr)
 }
 
+// Transform the homeserver ports into the base URL and federation base URL.
```

Comment:
It's worth noting that the convention for doc-comments in go is to start with the name of the thing you're describing:

```
// endpoints transforms the homeserver ports into the base URL and federation base URL.
```

I'll not insist you change everything, but if you could bear it in mind when you're fixing up doc-comments, that would be good.